chain/validation/applier.go

```diff
@@ -120,17 +120,26 @@ func (a *Applier) ApplyTipSetMessages(epoch abi.ChainEpoch, blocks []vtypes.Bloc
 }
 
 type randWrapper struct {
-	rnd vstate.RandomnessSource
+	rand vstate.RandomnessSource
 }
 
-func (w *randWrapper) GetRandomness(ctx context.Context, pers crypto.DomainSeparationTag, round abi.ChainEpoch, entropy []byte) ([]byte, error) {
-	return w.rnd.Randomness(ctx, pers, round, entropy)
+// TODO: these should really be two different randomness sources
+func (w *randWrapper) GetChainRandomness(ctx context.Context, pers crypto.DomainSeparationTag, round abi.ChainEpoch, entropy []byte) ([]byte, error) {
+	return w.rand.Randomness(ctx, pers, round, entropy)
+}
+
+func (w *randWrapper) GetBeaconRandomness(ctx context.Context, pers crypto.DomainSeparationTag, round abi.ChainEpoch, entropy []byte) ([]byte, error) {
+	return w.rand.Randomness(ctx, pers, round, entropy)
```

Comment:
This is going to prevent the tests from catching some errors. I believe we'll need to get chain-validation to pass us _two_ random sources.